### ISSUE TITLE
error loading new files when whitespace in filename

### ISSUE THREAD

**Issue by brvoigt** [dagelabi/TouchDesigner-Slideshow#1]
When filenames have a space in them an error occurs in /slideshow/loader/archive_manager/add_items 
This is due to going from s_items DAT to a CHOP format and then back to DATs in chopto1 DAT. This introduces an "_" for all spaces and the add_items script fails because it can not find a file of that name.

**Comment by makingpippop** [comment]
Yes indeed, for now, spaces or any special character in the filename will cause an error. I'm planning to update my "archive manager" to make those work.
